src/components/Form/Location/Layouts.js

```diff
@@ -14,5 +14,6 @@ export default {
     'US City, State, Zipcode International city',
   CITY_STATE_COUNTRY: 'City, State, Country',
   US_ADDRESS: 'US Address',
-  STREET_CITY: 'Street, City'
+  STREET_CITY: 'Street, City',
+  OFFENSE: 'US City, State, Zipcode, International city country'
```

Comment:
Would it be worth naming this more generically like the others in case this same layout ever needs to be used elsewhere in a non-offense context?

Reply:
Probably!